.gitignore

```diff
@@ -1 +1,168 @@
-node_modules
+
```

Comment:
I'd rather keep the lean one, if this PR requires extra ignore, please add those specifically

Reply:
alright